_posts/2016-05-20-tyxml4.md

```diff
@@ -0,0 +1,69 @@
+---
+title: TyXML 4.0.0
+layout: default
+author: The Ocsigen Team
+authorurl: https://ocsigen.org
+---
+
+This is with great pleasure that we are announcing the release of [TyXML 4.0.0][changelog]. This new version is focused on improving the user experience, both for old and new TyXML users.
```

Comment:
I would prefer:

- *It* is with great pleasure
- *The* new version
- for both old and new